packages/components/src/components/tile/tile.tsx

```diff
@@ -69,7 +69,7 @@ export class Tile extends LitElement implements InteractiveComponent, Selectable
    *
    * When `true`, renders without a border and padding for use by other components.
    *
-   * @deprecated No longer necessary.
+   * @deprecated  in v2.6.0, removal target v6.0 - No longer necessary.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated  in v2.6.0, removal target v6.0.0 - No longer necessary.
```